docs/articles_en/openvino-workflow/model-optimization-guide/weight-compression.rst

```diff
@@ -52,11 +52,12 @@ Compress Model Weights
 **8-bit weight quantization** method offers a balance between model size reduction and
 maintaining accuracy, which usually leads to significant performance improvements for
 Transformer-based models. Models with 8-bit compressed weights are performant on the
-vast majority of supported CPU and GPU platforms.
+vast majority of supported CPU and GPU platforms. By default, weights are compressed
+asymmetrically to "INT8_ASYM" mode with a non-fixed zero point.
```

Comment:
I would remove "with a non-fixed zero point" here as it can look confusing for those who are not aware of the problems we have in representation. Asymmetric quantization always means "non-fixed zero point".